devel xrdp_encoder gfx_wiretosurface2 ERROR

### xrdp version

devel 95bfb349a8833ea51c1932df6e4c17445131e61c

### Detailed xrdp version, build options

~~~
PASTE HERE
~~~


### Operating system & version

debian

### Installation method

git clone & make install

### Which backend do you use?

_No response_

### What desktop environment do you use?

_No response_

### Environment xrdp running on

VM

### What's your client?

mstsc and freerdp

### Area(s) with issue?

Crashes such as segfault

### Steps to reproduce

95bfb349a8833ea51c1932df6e4c17445131e61c
Just an log show error. Does not affect usage. 

After using the parameters ./configure --enable-fuse --enable-rfxcodec --enable-pixman --enable-sound --enable-ipv6 --enable-rdpsndaudin --with-imlib2 --with-freetype2 --enable-utmp --enable-devel-logging
log show errors occur after the connection session.


[2024-04-06T08:43:23.629+0800] [ERROR] [gfx_wiretosurface2(xrdp_encoder.c:779)] gfx_wiretosurface2: tiles_compressed 50 total_tiles 50 tiles_written 50
[2024-04-06T08:43:24.553+0800] [INFO ] [lib_mod_process_message(xup.c:1768)] lib_mod_process_message: type 3 len 505
[2024-04-06T08:43:24.376+0800] [ERROR] [gfx_wiretosurface2(xrdp_encoder.c:779)] gfx_wiretosurface2: tiles_compressed 52 total_tiles 52 tiles_written 52
[2024-04-06T08:43:24.533+0800] [INFO ] [lib_mod_process_message(xup.c:1768)] lib_mod_process_message: type 3 len 489
[2024-04-06T08:43:24.682+0800] [ERROR] [gfx_wiretosurface2(xrdp_encoder.c:779)] gfx_wiretosurface2: tiles_compressed 50 total_tiles 50 tiles_written 50
[2024-04-06T08:43:24.209+0800] [INFO ] [lib_mod_process_message(xup.c:1768)] lib_mod_process_message: type 3 len 489
[2024-04-06T08:43:24.223+0800] [ERROR] [gfx_wiretosurface2(xrdp_encoder.c:779)] gfx_wiretosurface2: tiles_compressed 50 total_tiles 50 tiles_written 50
[2024-04-06T08:43:24.747+0800] [INFO ] [lib_mod_process_message(xup.c:1768)] lib_mod_process_message: type 3 len 489
[2024-04-06T08:43:24.756+0800] [ERROR] [gfx_wiretosurface2(xrdp_encoder.c:779)] gfx_wiretosurface2: tiles_compressed 50 total_tiles 50 tiles_written 50
[2024-04-06T08:43:24.890+0800] [INFO ] [lib_mod_process_message(xup.c:1768)] lib_mod_process_message: type 3 len 673
[2024-04-06T08:43:24.898+0800] [ERROR] [gfx_wiretosurface2(xrdp_encoder.c:779)] gfx_wiretosurface2: tiles_compressed 31 total_tiles 31 tiles_written 31


### ✔️ Expected Behavior

_No response_

### ❌ Actual Behavior

_No response_

### Anything else?

_No response_

--- a/xrdp/xrdp_encoder.c
+++ b/xrdp/xrdp_encoder.c
@@ -773,7 +773,7 @@ gfx_wiretosurface2(struct xrdp_encoder *self,
         {
             break;
         }
-        LOG_DEVEL(LOG_LEVEL_ERROR, "gfx_wiretosurface2: "
+        LOG_DEVEL(LOG_LEVEL_INFO, "gfx_wiretosurface2: "
                   "tiles_compressed %d total_tiles %d tiles_written %d",
                   tiles_compressed, total_tiles,
                   tiles_written);

